theory/energetic sources/AGNs.md

```diff
@@ -7,3 +7,35 @@ nav_order: 1
 ---
 
 ## Active Galactic Nuclei (AGNs)
+
+### Introduction
+
+Active Galactic Nuclei or AGNs as they are known, are very powerful and bright galactic centers, powered by supermassive [black holes](../special%20stars/black%20holes.html). You can think of these black holes as cosmic vacuum cleaners of enormous size, pulling the surrounding gas and dust toward themselves.
+
+This material, as it approaches the black hole, begins to heat up due to friction and then radiates light, emitting energy across the whole spectrum-from radio waves to X-rays. This is the reason why AGNs are so bright; sometimes they outshine all the stars in their host galaxy put together. They are mostly situated at the center of the host galaxy.
```

Comment:
Please hyperlink black hole in this paragraph and continue to do so for the rest of the files in the PR, not just for black holes but for the rest too

Reply:
Resolved now